plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQuerySessionProperties.java

```diff
@@ -32,6 +35,7 @@ public final class BigQuerySessionProperties
     private static final String VIEW_MATERIALIZATION_WITH_FILTER = "view_materialization_with_filter";
     private static final String QUERY_RESULTS_CACHE_ENABLED = "query_results_cache_enabled";
     private static final String CREATE_DISPOSITION_TYPE = "create_disposition_type";
+    private static final String READ_PARALLELISM = "read_parallelism";
```

Comment:
I see that all the session properties are missing from `bigquery.md` - i'm guessing that the documentation can be done in a follow-up. Pls create a follow-up docs ticket for this.